packages/gatsby-plugin-google-analytics/src/utils.js

```diff
@@ -0,0 +1,13 @@
+const createFunctionWithTimeout = (callback, opt_timeout) => {
```

Comment:
We're not passing in opt_timeout anywhere so I think we can just ignore the parameter. Maybe just add it to the src/index file itself?

```suggestion
const createFunctionWithTimeout = (callback) => {
```